pkg/webhook/resources/virtualmachine/validator.go

```diff
@@ -296,3 +307,41 @@ func (v *vmValidator) checkReservedMemoryAnnotation(vm *kubevirtv1.VirtualMachin
 func (v *vmValidator) checkStorageResourceQuota(vm *kubevirtv1.VirtualMachine, oldVM *kubevirtv1.VirtualMachine) error {
 	return v.rqCalculator.CheckStorageResourceQuota(vm, oldVM)
 }
+
+func checkMaintenanceModeStrategyIsValid(newVM, oldVM *kubevirtv1.VirtualMachine) error {
+	labels := newVM.ObjectMeta.Labels
+	newStrategy, ok := labels[util.LabelMaintainModeStrategy]
+	if ok {
```

Comment:
consider to use following to reduce the code nesting
```
if !ok {
  return ...
}
// continue with ok
```

Reply:
Good idea. Done